smooth/components/component.py

```diff
@@ -101,6 +106,24 @@ def update_costs(self, results, sim_params, this_dependant_value=0):
         if self.artificial_costs is not None:
             self.results['art_costs'][sim_params.i_interval] = this_dependant_value * self.artificial_costs
 
+    def update_var_emissions(self, results, sim_params, this_dependant_value=0):

```

Comment:
Should we renamie update_costs to update_variable_costs so we are more analouge to update_var_emissions? and in update_financials and update_emissions we have then all the fix costs and opex

Reply:
https://github.com/rl-institut/smooth/pull/15/commits/7379fb862576e100f93c44a5556a00fa48df074e